README.md

```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
 
 ## Installation
 
-* `git clone <repository-url>` this repository
-* `cd @ember/optional-features`
-* `yarn install`
+```bash
+ember install @ember/optional-features
+```
 
-## Running
+## Usage
```

Comment:
Probably should mention this is intended for toggling Ember features in _apps_ (not addons)

Reply:
I added this detail and a quick description in place of the original default about stuff.